faq/index.md

```diff
@@ -66,7 +66,14 @@ alternative ways of requesting certain operations (e.g., resource creation via
 
 ## Is there a JSON Schema describing JSON API? <a href="#is-there-a-json-schema-describing-json-api" id="is-there-a-json-schema-describing-json-api" class="headerlink"></a>
 
-Not currently, no. JSON Schema cannot fully represent the semantics of JSON API, and so any such schema would be partial anyway.
+Yes, you can find the JSON Schema definition at
+[http://jsonapi.org/schema](http://jsonapi.org/schema). Please note that this
+schema is not a perfect document. Just because a JSON document may validate
+against this schema, that does not necessarily mean it is a valid JSON API
+document. The schema is provided for a base level sanity check.
```

Comment:
Would you mind doing that @eneuhauser and pinging back when ready? I'd love to get this merged but it would be nice for it to be a bit more clear.